### ISSUE TITLE
Unhandled Exception: System.FormatException: Input string was not in a correct format

### ISSUE THREAD

**Issue by pamiller3** [JamieH/TechnicSolderAutomate#3]
I setup the file as instructed but I am not sure if the mods I put into the mods folder was in the correct setup.  I have the .jars and I zipped them and placed that into the mods folder and still got the same result.  Is there a bug or am I not getting something?

**Comment by JamieH** [comment]
You need to start it like the following:
TechnicSolderAutomate.exe modsin modsout http://mysolder.solder.co.uk

**Comment by pamiller3** [comment]
Alright Jamie, maybe I need to elaborate more.  I am able to run your program without a hitch, but its spits out an error.  Here is the issue:  http://pastebin.com/yRELcaxX

Now as I have mentioned, I believe the issue is that I am not placing the mods into the inmods folder in the correct format.  I have tried several ways:

1) /inmods/fastcraft/mods/fastcraft-1.21.jar
2) /inmods/fastcraft-1.21.jar
3) /inmods/fastcraft-1.21.zip  <- .zip's archived as /mods/fastcraft-1.21.jar

I know I don't have the same directory names, but I switched it to yours real fast to test and have the exact same error.  I really appreciate your software as I know it is going to save me TONS of time, but I am just struggling to get it to work.  And thank you so much for your time!
